docs/guide/models/alchemical_network.rst

```diff
@@ -29,12 +29,14 @@ information for each :class:`.ChemicalSystem`, the :class:`.Transformation`
 also contains a :class:`.Protocol` and, when relevant, atom mapping
 information for alchemical transformations.
 
-A :class:`.ChemicalSystem` is made up of one or more
-:class:`gufe.ChemicalComponent`\ s. Each component represents a conceptual
-part of the total molecular system, e.g., ligand, or protein, or solvent.
-This allows us to easily identify what is changing between two nodes -- for
-example, a RBFE edge for ligand binding should have the same solvent and
-protein components, but different ligand components.
+A :class:`.ChemicalSystem` is made up of one or more ``ChemicalComponent``
+s. Each component represents a conceptual part of the total molecular
+system. A ligand would be represented by a :class:`.SmallMoleculeComponent`.
```

Comment:
We might need to keep in mind to come back here and fix this up when we can handle multiple small molecule components (i.e. in v0.8). Maybe add a todo?

Reply:
Phrasing here was very careful: "a ligand would be represented by..." doesn't change, even if other things are also represented by a SMC. 😉